src/test/java/io/vertx/tests/impl/ParseUtilsTest.java

```diff
@@ -1,5 +1,7 @@
-package io.vertx.httpproxy.impl;
+package io.vertx.tests.impl;
 
+import io.vertx.httpproxy.impl.ParseUtils;
+import junit.framework.TestCase;
```

Comment:
This looks like a wrong import, I think it should be `import static org.junit.Assert.assertEquals`